docs/developer-docs/integrations/ledger/ledger-local-setup.md

```diff
@@ -1,7 +1,18 @@
 # ICP ledger local setup
 
 ## Overview
-If you are working in a local development environment, i.e with a local replica instead of the public Internet Computer, you can't access the ICP ledger. In order to test your application that integrates with the ICP ledger locally, you need to deploy a local ledger canister. However, this local ledger canister won't have the history and balances of the live ICP ledger.
+If you are working in a local development environment, i.e with a local replica instead of the public Internet Computer, you can't access the ICP ledger. In order to test your application that integrates with the ICP ledger locally, you need to deploy a local ledger canister. However, this local ledger canister won't have the history and balances of the live ICP ledger. 
+
+:::info
+There are two ways of deploying an ICP ledger locally.
+- Use [dfx-nns](/docs/references/cli-reference/dfx-nns.md) to deploy the entire nns locally. Since the ICP ledger is part of the NNS, this command will also instlal an ICP ledger with canister ID `ryjl3-tyaaa-aaaaa-aaaba-cai`. This solution is fast and strait forward but also more heavy weight. 
```

Comment:
```suggestion
- Use [dfx-nns](/docs/references/cli-reference/dfx-nns.md) to deploy the entire NNS locally. Since the ICP ledger is part of the NNS, this command will also install an ICP ledger with canister ID `ryjl3-tyaaa-aaaaa-aaaba-cai`. This solution is fast and straight forward, but also more heavyweight. 
```